app/controllers/robot_verifications_controller.rb

```diff
@@ -18,7 +18,10 @@ def submission
     end
 
     if errors.empty?
-      @all_labels = @robot_verification.expected_layout(@batch, barcode_hash[:destination_plate_barcode])
+      @dest_plates, @source_plates, @ctrl_plates = @robot_verification.expected_layout(
```

Comment:
Splitting these up here makes the template simpler. @all_labels wasn't used any where else.